app/commands.py

```diff
@@ -135,7 +135,12 @@ def purge_functional_test_data(user_email_prefix):
             services = dao_fetch_all_services_by_user(usr.id)
             for service in services:
                 print(f"Deleting service {service.id=} {service.name=} that {usr.id} belongs to")
-                delete_service_and_all_associated_db_objects(service)
+                # This also deletes the functional if the test users are associated
+                # It may require to create essential db object to run functional tests
+                if service.name != "GOV.UK Notify":
```

Comment:
we can check `NOTIFY_SERVICE_ID` in the config - feels a bit more robust to check that rather than the service name (that UUID is hard coded from an alembic migration)